### PR TITLE
Minor comment + formatting improvement

### PR DESCRIPTION
**Pull request by colonelsalt**
First-time contributor trying my hand at making a pull request.

Added file description for skelmdl.h + fixed formatting inconsistencies for if-elses and switch-cases in weapon.cpp.

### CODE REVIEW

File: inexor/fpsgame/weapon.cpp [hunk 1]
```diff
@@ -711,7 +711,6 @@ namespace game
                 break;
 
             case GUN_SG:
-            {
```
Comment (Fohlen) [inline inexor/fpsgame/weapon.cpp]: what's up with the formatting? Looks like we really need #417

**Comment by Fohlen** [comment]
Hey @colonelsalt, welcome to Inexor, and thank you very much for your contribution!
It looks sane from here, and I am sure that we will include it soon, maybe `squash-merge` together with #466 @a-teammate ?

**Comment by IAmNotHanni** [comment]
Hello @colonelsalt. Welcome to Inexor!

File: inexor/fpsgame/weapon.cpp (hunk 1 above)
Comment (Croydon) [reply inexor/fpsgame/weapon.cpp]: Removing these brackets doesn't work. https://travis-ci.org/inexorgame/inexor-core/jobs/270150983#L6596

**Comment by Croydon** [comment]
@colonelsalt Welcome to Inexor :)

File: inexor/fpsgame/weapon.cpp (hunk 1 above)
Comment (bobbybrot) [reply inexor/fpsgame/weapon.cpp]: @Fohlen What is going on with #417 anyways, would you know?

**Comment by Croydon** [comment]
Are you still interested in doing this @colonelsalt ?

**Comment by Croydon** [comment]
@colonelsalt I'm taking this as a no. 

I have taken this over and fixed the commit. @Fohlen approve please?

**Pull request merged**